### DISCUSSION TITLE
age --help should write to stdout

### DISCUSSION THREAD

**Discussion by vbezhenar** [FiloSottile/age#434] [UX reports]
I think most utilities write help text to stdout. It helps to use something like `age --help | less`. Right now I have to use `age --help 2>&1 | less`

**Comment by FiloSottile** [comment]
I generally agree with your preference, but how the flags package handles the help text is that it's printed to the same place for `--help` and for a wrong flag (which definitely needs to go to stderr). The preference doesn't feel worth reimplementing `--help`/`-h` handling, although it's definitely doable.

**Reply by alerque** [reply]
This sounds like something that should be fixed in the "flags package" then, whatever that is. Handling errors vs. output is pretty standard stuff and sending help text to STDOUT is pretty much universally considered better (not 100% agreement of course, there are quite a few utilities that don't do this). At the very least it should be a relatively easy to enable option in and flag handling package.
